shared/markdown/__test__/parser.test.js

```diff
@@ -124,6 +130,31 @@ describe('Markdown parser', () => {
   })
   it('parses native zwj emoji correctly', () => {
     check('👩‍❤️‍💋‍👩 👩‍👩‍👧‍👧!')
+    check('👩‍❤️‍💋‍👨 👨‍👧 👨‍👦 👨‍👩‍👧‍👦 👨‍👨‍👧‍👧 👩‍👩‍👧‍👦 👩‍❤️‍👩')
+  })
+  it('parses qualified and non-qualified emoji identically', () => {
+    // Check that the trees are identical
+    // and contain the same emoji
+    const verify = (asts, emojiString) => {
+      asts.forEach(ast => {
+        expect(ast.type).toBe('markup')
```

Comment:
i think you can replace this entire block with something like

```
expect(ast).toEqual({
  type: 'markup',
  children: [{
    type: 'text-block',
    children: [{
      type: 'native-emoji',
      children: [emojiString],
    }]
  }]
})
```
and maybe inline it below